base/docs/helpdb/Base.jl

```diff
@@ -10485,3 +10485,10 @@ to. This is useful when writing custom `serialize` methods for a type, which opt
 data written out depending on the receiving process id.
 """
 Base.worker_id_from_socket
+
+"""
+    Base.cluster_cookie([cookie]) -> cookie
+
+Returns the cluster cookie. If a cookie is passed, also sets it as the cluster cookie.
+"""
+Base.cluster_cookie
```

Comment:
Had it inline but moved it after removing the export since we cannot have the function definition as `Base.cluster_cookie`.